spec/ics-2-consensus-verification/README.md

```diff
@@ -0,0 +1,310 @@
+---
+
+ics: 2
+title: Consensus Verification
+stage: draft
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-04-02
+
+---
+
+## Synopsis
+
+This standard specifies the properties that consensus algorithms of chains implementing IBC are
```

Comment:
It also specifies the algorithms for creating a client instance now, right?

Reply:
Yes, updated